controls/nist_ocp4.yml

```diff
@@ -689,11 +689,21 @@ controls:
     https://issues.redhat.com/browse/CMP-216
   rules: []
 - id: AC-4(8)
-  status: pending
+  status: not applicable
   notes: |-
-    A control response is planned. Progress can be tracked via:
+    Security policy filters for data structures can check for maximum file lengths,
+    maximum field sizes, and data/file types (for structured and unstructured data)
+    can be met by OpenShift.
+
+    All of OpenShift extends the k8s api with custom resource defintions.
+    The api has built-in limits and there are additional validations we can perform. [1]
+
+    However security policy filters for data content can check for specific words enumerated values
```

Comment:
Is the first sentence missing a "that" or a "which"? Were you planning to write something  like "security policy filters for data content which check for specific words, enumerated values...are an organizational control"